samples/client/petstore/typescript-angular2/npm/api/PetApi.ts

```diff
@@ -308,7 +308,11 @@ export class PetApi {
         let queryParameters = new URLSearchParams();
         let headers = new Headers(this.defaultHeaders.toJSON()); // https://github.com/angular/angular/issues/6845
         if (status !== undefined) {
-            queryParameters.set('status', <any>status);
+            if(status){
```

Comment:
Can we do `if (status) {` instead so as to conform to the TS style guide: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#style ?